app/mixins/setup-submit-controller.js

```diff
@@ -21,7 +22,11 @@ export default Ember.Mixin.create({
                 controller.set('providers', providers);
             }
         );
-
+        this.get('theme.provider').then(provider => {
+            provider.query('licensesAcceptable', {'page[size]': 20}).then(licenses => {
```

Comment:
Yep, the preprint provider populate script doesn't contain license information yet